### PR TITLE
Radium to emotion

### PR DESCRIPTION
**Pull request by carlospaelinck**
- [x] Convert `@Radium` decorators for components to `react-emotion` components
- [x] Refactor `Slide` transitions and simplify logic
- [x] Update tests

### CODE REVIEW

**Comment by ebertmi** [comment]
What is the reasoning behind switching from radium to emotion?

**Comment by carlospaelinck** [comment]
@ebertmi This is the first phase in rewriting the themeing story and also a general refactor. We want to move to a modern css-in-js styling solution. Then look at where things are in how to make theming more approachable.

File: src/components/text.js [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+    padding: '0',
+    lineHeight,
+    transform: `scale(${scale})`,
+    transformOrigin: 'center top'
+  },
+  styles.typeface,
+  styles.user
+]);
+
+const UnfitText = styled.p(({ lineHeight, styles }) => [
```
Comment (kenwheeler) [inline src/components/text.js]: THIS TEXT IS UNFIT

**Comment by kenwheeler** [comment]
OK hold up, got some prollems. I'll send you videos

**Pull request merged**